### PR TITLE
Request to Add new ARM Template: dsc-extension-azure-automation-pullserver

### PR DESCRIPTION
**Pull request by charlesjoy**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@charlesjoy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: dsc-extension-azure-automation-pullserver/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        }
+    },
+    "variables": {
+
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Compute/virtualMachines/extensions",
+            "name": "[concat(parameters('vmName'),'/', 'dscExtension')]",
+            "apiVersion": "2014-12-01-preview",
```
Comment (singhkay) [inline dsc-extension-azure-automation-pullserver/azuredeploy.json]: I believe this API is deprecated for Microsoft.Compute. Please move to 2015-06-15

**Comment by singhkay** [comment]
@charlesjoy Is there a tutorial on how to setup the dependencies that are needed before this template can be executed? Does one just need to setup a vanilla VM or something more is needed?

**Comment by charlesjoy** [comment]
@singhkay Yes, all that is needed from the Azure IaaS side is a vanilla VM. From an Azure Automation DSC perspective, what I put in the README.md file (http://aka.ms/DSCLearnMore) is what the Azure Automation DSC team has stated is the best set of tutorial level instructions for pre-requisites for connection to that service.

**Comment by azurecla** [comment]
__@charlesjoy__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

**Pull request merged**